src/maps/collectionFlow.ts

```diff
@@ -0,0 +1,19 @@
+import { CollectionFlow, CollectionFlowResponse } from '@/models'
+import { MapFunction } from '@/services/Mapper'
+
+export const mapCollectionFlowResponseToCollectionFlow: MapFunction<CollectionFlowResponse, CollectionFlow> = function(source) {
+  return {
+    collectionType: 'flow',
+    name: source.name,
+    description: source.description.summary,
+    returns: source.description.returns,
+    examples: source.description.examples,
+    documentationUrl: source.documentation_url,
+    installCommand: source.install_command,
+    logoUrl: source.logo_url,
+    parameters: source.parameters,
+    pathContainingFlow: source.path_containing_flow,
```

Comment:
I'm using this field for the deployment create POST endpoint, which expects it as simply `'path'`

```suggestion
    path: source.path_containing_flow,
```

Reply:
I like that more anyways